Description: fix laptop build-in touchpad quick double click not working
 防抖动功能过滤掉了间隔时间比较短的双击事件，
 在这个触摸板上禁用掉防抖动功能。
Author: quezhiyong <quezhiyong@uniontech.com>


Origin: https://gerrit.uniontech.com/plugins/gitiles/base/libinput/+/cbdf940f9dee5bc67e953b2834c5a580ff0585d3
Bug: https://pms.uniontech.com/zentao/bug-view-85625.html
Last-Update: 2022-05-19

--- libinput-1.19.2.orig/quirks/30-vendor-logitech.quirks
+++ libinput-1.19.2/quirks/30-vendor-logitech.quirks
@@ -46,6 +46,11 @@ MatchVendor=0x046D
 MatchProduct=0x4011
 AttrPalmPressureThreshold=400
 
+# laptop built-in touchpad
+[ImPS/2 Logitech Wheel Mouse]
+MatchName=ImPS/2 Logitech Wheel Mouse
+ModelBouncingKeys=1
+
 [Logitech MX Master]
 MatchVendor=0x46D
 MatchProduct=0x4041
